hack/testing/prep-host

```diff
@@ -32,3 +40,6 @@ do
 done
 
 export PATH=/usr/local/bin:$PATH
+
+sudo sed -i -e 's/--selinux-enabled//g'  /etc/sysconfig/docker
```

Comment:
@stevekuznetsov  found out that removing this from the docker config allowed me to pass my server start problem